packages/app/src/components/PageEditor/CodeMirrorEditor.jsx

```diff
@@ -668,6 +664,14 @@ export default class CodeMirrorEditor extends AbstractEditor {
     );
   }
 
+  renderEmojiPickerModal() {
+    return (
+      <Modal isOpen={this.state.isEmojiPickerShown} toggle={this.toggleEmojiPicker} onOpened={this.emojiPickerOpened}>
+        <Picker set="apple" autoFocus style={{ width: '100%', position: 'absolute' }} />
+      </Modal>
+    );
+  }
```

Comment:
Please just show emojiPicker not as a modal :D

Reply:
updated : https://github.com/weseek/growi/pull/5123/commits/047f6d97431e48392a0738b5ed53a18268acf94d